src/test/java/org/tikv/raw/RawKVClientTest.java

```diff
@@ -95,7 +96,24 @@ public void tearDown() throws Exception {
 
   // tikv-4.0 does not support atomic api
   @Ignore
-  public void atomicAPITest() {
+  public void rawCASTest() {
+    if (!initialized) return;
+    long ttl = 10;
+    ByteString key = ByteString.copyFromUtf8("key_atomic");
+    ByteString value = ByteString.copyFromUtf8("value");
+    ByteString value2 = ByteString.copyFromUtf8("value2");
+    client.delete(key);
+    client.compareAndSet(key, null, value, ttl);
+    try {
+      client.compareAndSet(key, null, value2, ttl);
+    } catch (RawCASConflictException err) {
+      assert err.getPrevValue() == Optional.of(value);
+    }
+  }
```

Comment:
```suggestion
  @Test
  public void rawCASTest() {
    if (!initialized) return;
    long ttl = 10;
    ByteString key = ByteString.copyFromUtf8("key_atomic");
    ByteString value = ByteString.copyFromUtf8("value");
    ByteString value2 = ByteString.copyFromUtf8("value2");
    client.delete(key);
    client.compareAndSet(key, Optional.empty(), value, ttl);
    Assert.assertEquals(value, client.get(key).get());
    try {
      client.compareAndSet(key, Optional.empty(), value2, ttl);
      Assert.fail("compareAndSet should fail.");
    } catch (RawCASConflictException err) {
      Assert.assertEquals(value, err.getPrevValue().get());
    }
  }
```

We'd better make use of `org.junit.Assert`. At least for new test cases.